bug-bounty-hof/web.yml

```diff
@@ -1,4 +1,69 @@
 names:
+- name: Griffin Francis
+  date: 2022-03-28
+- name: zy9ard3
+  date: 2022-03-22
+  twitter: "@zy9ard3"
+  url: https://twitter.com/@zy9ard3
+- name: Jun Kokatsu
+  date: 2022-03-22
+  twitter: "@shhnjk"
+  url: https://twitter.com/@shhnjk
+- name: Codermak
+  date: 2022-03-22
+  twitter: "@arshadkazmi42"
+  url: https://twitter.com/@arshadkazmi42
+- name: zeb0x01
+  date: 2022-03-15
+  url: https://twitter.com/https://www.linkedin.com/in/mehboob-shaikh-7791a3169
+- name: Anonymous
+  date: 2022-03-15
```

Comment:
Hello @tomrittervg , how can I know who this Anonymous entry from? I checked the bugs and I didn't find this

Reply:
reporter asked to be anonymous